lib/autoscaling.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const ApiGroup = require('./api-group');
+const Deployments = require('./deployments');
+
+class Autoscaling extends ApiGroup {
+  constructor(options) {
+    const commonResources = [
+      'horizontalpodautoscalers'
+    ];
+    options = Object.assign({}, options, {
+      path: 'apis/autoscaling',
+      version: options.version || 'v1',
+      groupResources: commonResources.concat(['deployments']),
```

Comment:
You'd only need `deployments` here if there was a `/apis/autoscaling/v1/deployments` resource, which there isn't (nor is there a `/apis/autoscaling/v1/namespace/:namespace/deployments` resource.

Remove the deployments.

Reply:
Yikes, that's unexpected.  I think this works for me:

```js
'use strict';

const ApiGroup = require('./api-group');

class Autoscaling extends ApiGroup {
  constructor(options) {
    const commonResources = [
      'horizontalpodautoscalers'
    ];
    options = Object.assign({}, options, {
      path: 'apis/autoscaling',
      version: options.version || 'v1',
      groupResources: commonResources,
      namespaceResources: commonResources
    });
    super(options);
  }
}

module.exports = Autoscaling;
```

Reply:
Updated with code sample...